_maps/map_files/generic/CentCom_skyrat_z2.dmm

```diff
@@ -6436,7 +6437,12 @@
 	dir = 8
 	},
 /obj/effect/turf_decal/trimline/blue/filled/line,
-/obj/machinery/vending/autodrobe,
+/obj/machinery/vending/imported{
```

Comment:
you don't need to manually set the prices on these. All vending machines on other z-levels have their prices zeroed.